src/main/kotlin/org/neo4j/graphql/GraphQLExtensions.kt

```diff
@@ -85,3 +86,10 @@ fun paramName(variable: String, argName: String, value: Any?) = when (value) {
     is VariableReference ->  value.name
     else -> "$variable${argName.capitalize()}"
 }
+
+fun List<FieldDefinition>.findID(): FieldDefinition? = this.find { it.type.name() == "ID" }
```

Comment:
don't put extension functions on List<X> if it's avoidable but on the object type instead